jetty-server/src/main/java/org/eclipse/jetty/server/handler/InetAccessHandler.java

```diff
@@ -188,13 +188,16 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
     protected boolean isAllowed(InetAddress addr, Request baseRequest, HttpServletRequest request)
     {
         String name = baseRequest.getHttpChannel().getConnector().getName();
+        boolean includedConnectorName = _names.isIncluded(name);
+        boolean excludedConnectorName = _names.isExcluded(name);
+        boolean filterAppliesToThisConnector = (_names.includesSize() == 0 || includedConnectorName) && !excludedConnectorName;
         if (LOG.isDebugEnabled())
         {
-            Boolean allowedByName = _names.isIncludedAndNotExcluded(name);
             Boolean allowedByAddr = _addrs.isIncludedAndNotExcluded(addr);
-            LOG.debug("{} allowedByName={} allowedByAddr={} for {}/{}", this, allowedByName, allowedByAddr, addr, request);
+            LOG.debug("{} includedConnectorName={} excludedConnectorName={} filterAppliesToThisConnector={} allowedByAddr={} for {}/{}",
```

Comment:
leave the  "allowedBy____" syntax.
We don't want to get in the habit of dumping every little include/exclude for every possible path through this.
That's what the `.dump()` methods are for.

Reply:
Hi. I went ahead and dumped in something that definitely works for now. Will get back to the cleaner version once i figure out why it doesn't work.

Reply:
OK i made some changes that work for me that still use this `IncludeExcludeSet` class. please review.